modules/post/multi/gather/aws_ec2_instance_metadata.rb

```diff
@@ -50,14 +50,14 @@ def check_curl
 
   def get_aws_metadata(base_uri, base_resp)
     r = {}
-    base_resp.split(/\r\n/).each do |l|
-      new_uri = base_uri.merge("./#{l}")
-      if l =~ %r{/$}
+    base_resp.split(/\n/).each do |l|
+      new_uri = "#{base_uri}#{l}"
+      if l =~/\/$/
```

Comment:
`if l =~ %r{/$}` and `if l =~/\/$/` are functionally identical.

Not sure why this was changed? `%r{}` notation is preferred over escaping `/`.

However, if `l` is only ever going to be one line, then `l.end_with?('/')` could be used instead, and is preferred.